src/test/java/core/basesyntax/service/FileReaderServiceTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class FileReaderServiceTest {
+    @Test(expected = RuntimeException.class)
+    public void readFromDoesNotExistFile_NotOk() {
+        new FileReaderServiceImpl().readFromFile("src/does_not_exist_file.csv");
+    }
+
```

Comment:
You initialize your FileReader twice in this test class. Instead you can use the special method @BeforeClass to do it only once.